holoviews/plotting/bokeh/chart.py

```diff
@@ -264,6 +264,9 @@ class SpikesPlot(PathPlot):
     position = param.Number(default=0., doc="""
       The position of the lower end of each spike.""")
 
+    show_legend = param.Boolean(default=True, doc="""
```

Comment:
It's always been there I'm just being explicit about which plots have it enabled and disabled by default.